docs/installation/Running-with-Docker.md

```diff
@@ -104,6 +106,14 @@ It should be mentioned that if your dataset is very large, please adjust the `-X
 
 Note, `.osm`, `.osm.gz`, `.osm.zip` and `.pbf` file format are supported as OSM files.
 
+### Using pre-built graphs and ors.war file
+
+Instead of building the routing graph based on the OSM file, it is also possible to use pre-built graphs along with a pre-compiled ors.war file.
+
+1. Set `USE_PREBUILT` to `True` and `BUILD_GRAPHS` to `True` in the `docker-compose.yml` file.
```

Comment:
I think there is a typo here: 

```
and BUILD_GRAPHS to True
```

should be 
```
and BUILD_GRAPHS to False
```

as `USE_PREBUILT` and `BUILD_GRAPHS` cannot be both `True`